pygdbmi/gdbcontroller.py

```diff
@@ -108,7 +108,7 @@ def write(self, mi_cmd_to_write,
             print('\nwriting: %s' % mi_cmd_to_write)
 
         if not mi_cmd_to_write.endswith('\n'):
-            mi_cmd_to_write_nl = mi_cmd_to_write + '\n'
+            mi_cmd_to_write = mi_cmd_to_write + '\n'
```

Comment:
I prefer this to be a new variable with the `_nl` suffix. Can an `else` statement be added here which does `mi_cmd_to_write_nl = mi_cmd_to_write`